python/scripts/meta.json

```diff
@@ -171,6 +171,178 @@
     ],
     "replace_table": false
   },
+  "census_tract_to_neighborhood_cluster": {
+    "cleaner": "CensusTractToNeighborhoodClusterCleaner",
+    "fields": [
+      {
+        "display_name": "Census tract",
+        "display_text": "Census tract",
+        "required_in_source": true,
+        "source_name": "Geo2010",
+        "sql_name": "census_tract",
+        "type": "text"
+      },
+      {
+        "display_name": "Neighborhood cluster description",
+        "display_text": "Neighborhood cluster description",
+        "required_in_source": true,
+        "source_name": "Cluster_tr2000",
+        "sql_name": "cluster_desc",
```

Comment:
for consistency with other tables, we should call this neighborhood_cluster_desc. Here is where we are documenting the column name conventions: https://github.com/codefordc/housing-insights/blob/dev/docs/resources/data-formatting.md